source/rules/literals.rs

```diff
@@ -722,78 +640,6 @@ mod tests {
     };
 
     #[test]
-    fn case_null() {
-        let input  = Span::new(b"null");
```

Comment:
Should all of these tests be moved? I am not sure if they are just redundant then, in particular since a test for unqualified names already exists: https://github.com/tagua-vm/parser/blob/ff0ca1f618fcce0fc7453d893af0f1dc55f93009/source/rules/tokens.rs#L242-L248